src/HtmlPurifier.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I've asked for clarification on the talk page of the Wikipedia CC: https://www.mediawiki.org/wiki/Manual_talk:Coding_conventions/PHP#Spaces_inside_declare.28.29